docs/core-concepts/packages.md

```diff
@@ -0,0 +1,143 @@
+Packages, Metapackages and Frameworks
+=====================================
+
+.NET Core is a platform of NuGet packages. Some product experiences benefit from fine-grained definition of packages while others from course-grained. To accomodate this duality, the product is distributed as a fine-grained set of packages and then described in courser chunks with a package type informally called "metapackage".
```

Comment:
is a platform **made** of NuGet packages?
accomodate -> accommodate
why do you say informally? do we have a formal name for that too?